i3lock.c

```diff
@@ -976,10 +977,16 @@ int main(int argc, char *argv[]) {
     /* Pixmap on which the image is rendered to (if any) */
     xcb_pixmap_t bg_pixmap = draw_image(last_resolution);
 
-    /* open the fullscreen window, already with the correct pixmap in place */
+    /* Open the fullscreen window, already with the correct pixmap in place */
     win = open_fullscreen_window(conn, screen, color, bg_pixmap);
     xcb_free_pixmap(conn, bg_pixmap);
 
+    cursor = create_cursor(conn, screen, win, curs_choice);
+
+    /* Setting to locking state while grabbing the pointer. */
```

Comment:
The comment describes what we can already read from the code. Could you make it more informative by describing the effects? E.g.: Display the “locking” message while trying to grab the pointer/keyboard